src/peft/utils/save_and_load.py

```diff
@@ -193,6 +213,20 @@ def set_peft_model_state_dict(model, peft_model_state_dict, adapter_name="defaul
             rank_pattern = config.rank_pattern
             if rank_pattern is not None:
                 model.resize_modules_by_rank_pattern(rank_pattern, adapter_name)
+        elif config.peft_type == PeftType.VERA:
```

Comment:
Good job checking all these conditions! Please break the long lines to 120 chars.